tests/unit/ServiceProviderTest.php

```diff
@@ -19,6 +19,24 @@ protected function getEnvironmentSetUp($app)
         $app['config']->set('kinesis.aws.key', 'key');
         $app['config']->set('kinesis.aws.secret', 'secret');
         $app['config']->set('kinesis.level', 1);
+
+
+        $app['config']->set('logging.channels', [
+            'kinesis' => [
+                'driver' => 'monolog',
+                'handler' => \PodPoint\KinesisLogger\Monolog\KinesisHandler::class,
+                'with' => [
+                    'streamName' => config('kinesis.stream'),
+                    'level' => config('kinesis.level'),
+                ],
+                'formatter' => \PodPoint\KinesisLogger\Monolog\KinesisFormatter::class,
+                'formatter_with' => [
+                    'name' => config('app.name'),
+                ],
+            ],
+        ]);
+
+        $app['config']->set('logging.default', 'kinesis');
```

Comment:
Looking at this with Andrew we found that the instructions in this package show that we will be using config file data inside other config files - for example kinesis.stream and kinesis.level data is relied on inside the config/logging file. This can cause an issue depending on which config files are loaded first. 

A solution can be to get rid of kinesis.php moving all necessary data within the kinesis object and then removing the config dependencies. 

'formatter_with' => [
        'name' => config('app.name'),
    ],

can become:
'formatter_with' => [
        'name' => env('app.name') ?? 'Laravel',
    ],

I think @oaklees sent more information about a different solution for this.